pocketbase/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: pocketbase_app_1
+      APP_PORT: 8090
+      PROXY_AUTH_ADD: "false"
+  
+  app:
```

Comment:
Can you please try out the app with this container running under `user: "1000:1000"`. Right now the container is running as `root`, but if we can reduce ownership/permissions to the `umbrel` user then this is ideal from an app sandbox point of view.

Reply:
I have tried using `user: 1000:1000`, but unfortunately I was not able to create collections within pocketbase. Then I have removed the user... line and it worked. To double check I have added it back and the error unfortunately came back

Reply:
Thanks for testing this @sharknoon 🤝. I have added a comment to document that pocketbase needs to be run with root permissions https://github.com/getumbrel/umbrel-apps/pull/1082/commits/ed35d230a4dde9dbba6ed66957dc97727d122469